WeatherApp/Models/AddressFormater.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  AddressFormater.swift
+//  WeatherApp
+//
+//  Created by Oleksii Kolomiiets on 5/25/18.
+//  Copyright © 2018 Oleksii Kolomiiets. All rights reserved.
+//
+
+import Foundation
+import MapKit
+
+class AddressFormater: FormatterProtocol {
+    
+    init(_ selectedItem: MKPlacemark) {
```

Comment:
Failable init looks better.
Return nil if required data is missing in `selectedItem`